test/libsolidity/semanticTests/salted_create/salted_create.sol

```diff
@@ -17,8 +17,15 @@ contract A {
     }
 }
 // ====
-// compileViaYul: also
 // EVMVersion: >=constantinople
+// compileViaYul: also
 // ----
 // different_salt() -> true
+// gas ir: 107186
+// gas legacy: 125934
+// gas legacyOptimized: 125869
 // same_salt() -> true
+// gas ir: 98439503
+// gas legacy: 98439401
+// gas legacyOptimized: 98439300
```

Comment:
The difference is surprisingly big here. Does try/catch in `same_salt()` really cost that much or is it due to a different salt?

Reply:
Not sure, but @chriseth should probably know